components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtValidator.java

```diff
@@ -77,6 +77,8 @@
             "for tenantDomain:%s.";
     private static final String CLAIM_NOT_AVAILABLE = "Local claim %s is not available in the server " +
             "for tenantDomain:%s.";
+    private static final String SP_CLAIM_NOT_AVAILABLE = "Application Claim URI '%s' is not defined " +
+            "for Service Provider:%s.";
```

Comment:
```suggestion
            "for application:%s.";
```
```suggestion
            "for Service Provider:%s.";
```

Reply:
addressed in b29cb4f92c6de08a226dd39ed245c1df3363f5cc